card/mod/carrierwave/set/all/file_utils.rb

```diff
@@ -0,0 +1,30 @@
+module ClassMethods
+  def update_all_storage_locations
+    Card.search(type_id: ["in", FileID, ImageID])
+        .each(&:update_storage_location!)
+  end
+
+  def delete_tmp_files_of_cached_uploads
+    draft_actions_with_attachment.each do |action|
+      # we don't want to delete uploads in progress
+      if older_than_five_days?(action.created_at) && (card = action.card)
+        # we don't want to delete uploads in progress
+        card.delete_files_for_action action
+        action.delete
+      end
+    end
+  end
+
+  def older_than_five_days? time
```

Comment:
I wrote that method instead of a comment about the 432000. Now I need the comment for the 432000 as default time :)